### ISSUE TITLE
[🐛 Bug]: Upgrade to Allure 2 breaks wdio-video-reporter

### ISSUE THREAD

**Issue by Jmcosel** [webdriverio/webdriverio#9930]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

8.5.9

### Node.js Version

18.15.0

### Mode

WDIO Testrunner

### Which capabilities are you using?

```typescript
{
  unhandledPromptBehavior: 'ignore',
  browserName: 'chrome'
}
```

### What happened?

Upon release of `@wdio/allure-reporter@8.5.0` (corresponding to PR #9704), it seems like the upgrade to using `allure-js-commons@2.0.0` has caused its compatibility with `wdio-video-reporter` to break.

According to the error received, it seems like support for the file type `video/mp4`, which is what `wdio-video-reporter` passes via the `addAttachment` method, does not exist any longer, so the framework entirely fails because it doesn't know what to do with this file type.

### What is your expected behavior?

The framework should know how to attach video files, as it did properly before the upgrade.

### How to reproduce the bug.

See the following gist:
https://gist.github.com/Jmcosel/98fe2548afa8a313ed0cc41ce0b08d63

### Relevant log output

```typescript
[0-0] Error in "Debug.Debug"
Error: Unrecognized extension: video/mp4
    at typeToExtension (/Users/jcosel/Documents/debug-wdio/node_modules/allure-js-commons/src/writers/utils.ts:35:9)
    at buildAttachmentFileName (/Users/jcosel/Documents/debug-wdio/node_modules/allure-js-commons/src/AllureRuntime.ts:12:36)
    at AllureRuntime.writeAttachment (/Users/jcosel/Documents/debug-wdio/node_modules/allure-js-commons/src/AllureRuntime.ts:46:22)
    at process.emit (node:events:513:28)
    at process.emit (node:domain:489:12)
    at process.emit.sharedData.processEmitHook.installedValue [as emit] (/Users/jcosel/Documents/debug-wdio/node_modules/@cspotcode/source-map-support/source-map-support.js:745:40)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
@epszaw does Allure support video like that?

**Comment by robbiexiong** [comment]
I also experience the same issue and error. It seems to work fine running locally on my box however, it just breaks on remote executions, so I am not sure why. 
This started after upgrading to webdriverio v8. Also updated
"@wdio/allure-reporter": "^8.4.0"
"wdio-video-reporter": "^3.5.0"
"@wdio/spec-reporter": "^8.4.0",

**Comment by KurnosovNikita** [comment]
The same issue for me:

"@cucumber/cucumber": "9.0.1",
"@wdio/allure-reporter": "8.6.8",
"@wdio/cli": "8.6.8",
"@wdio/cucumber-framework": "8.6.8",
"@wdio/local-runner": "8.6.8",
"@wdio/selenium-standalone-service": "8.6.8",
"@wdio/spec-reporter": "8.6.8",
"wdio-video-reporter": "3.5.0",
"webdriverio": "8.6.8",

```
reporters: [
    'spec',
    [
        video,
        {
            saveAllVideos: true, // If true, also saves videos for successful test cases
            videoSlowdownMultiplier: 5, // Higher to get slower videos, lower for faster videos [Value 1-100]
        },
    ],
    ['allure', {
        outputDir: './_results_/allure-raw',
        disableWebdriverStepsReporting: true,
        disableWebdriverScreenshotsReporting: true,
    }],
],
```

**Comment by christian-bromann** [comment]
We would appreciate any help on this so please help us out! Have a look at our [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) and reach out if you have any questions. Cheers!

**Comment by HedgU4** [comment]
The is in that under the hood new npm of allure-js-commons is installed(2.1.0) instead of an old deprecated version(1.3.2)
list of supported attachment types https://github.com/allure-framework/allure-js/blob/master/packages/allure-js-commons/src/writers/utils.ts
And there are no video types at all.

**Comment by NaZaRKIN123** [comment]
> And there are no video types at all.

There is a Webm. Which is video type.

**Comment by HedgU4** [comment]
> > And there are no video types at all.
> 
> There is a WebM. Which is a video type.

Thx for pointing. In this case, only a small fix to wdio-video-reporter is needed.

**Comment by HedgU4** [comment]
> > Thx for pointing. In this case, only a small fix to wdio-video-reporter is needed.
> 
> Do you like to take the initiative and provide a PR?

I guess Yes, but still working on a fix, since simply changing contentType to 'video/webm' does not throw the error but also doesn't attach a video file to report properly.
Is it OK if a fix will combine with migration to wdio v8?

**Comment by christian-bromann** [comment]
> Is it OK if a fix will combine with migration to wdio v8?

That would be awesome, thanks!

**Comment by HedgU4** [comment]
I didn't find a way how to fix it without breaking changes.
fix require new argument AttachmenOptions(allure) that contains fileExtention.
webm is not supported correctly =(

**Comment by HedgU4** [comment]
fix for wdio-video-report to use webm instead mp4
https://github.com/mbIkola/wdio-video-reporter/ [v3.6.3]
example of wdio config
```
    [video, {
      saveAllVideos: false,       // If true, also saves videos for successful test cases
      videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
      videoRenderTimeout: 5,      // Max seconds to wait for a video to finish rendering
      videoFormat: 'webm'
    }],
```

**Comment by HedgU4** [comment]
fixed in 4.0.1

**Comment by christian-bromann** [comment]
Awesome, closing then.
